panoramix/assets/javascripts/modules/panoramix.js

```diff
@@ -217,6 +217,15 @@ var px = (function () {
         others += widget.find('.slice_header').height();
         return widget.height() - others;
       },
+      setResizeOnWindowResize: function () {
+        var resizeTimer;
```

Comment:
this is basically to make sure that the function is called when the resizing is done, as opposed to calling the function a thousand times during the drag event